core_data_modules/traced_data/io.py

```diff
@@ -357,8 +357,19 @@ def import_coda_2_to_traced_data_iterable_multi_coded(cls, user, data, message_i
                 for label in labels:
                     assert label.scheme_id.startswith(scheme.scheme_id)
                     label.scheme_id = scheme.scheme_id
+                    
+                # De-duplicate the imported labels by selecting the first label with each code id.
+                # This is required in cases where the same label was applied to this message under different columns
+                # of the same code scheme, and is possible now that we have normalised the scheme ids.
+                deduplicated_labels = []
```

Comment:
Consider `unique_labels`

Reply:
Went with `unique_labels_by_code_id`, to be super clear about the condition for uniqueness we're using.